libs/handoversidesheet/src/lib/ui-sidesheet/HandoverSidesheet.tsx

```diff
@@ -224,6 +231,13 @@ const HandoverSidesheetComponent = (props: Required<HandoverProps>) => {
                 isLoading={isDataFetchingUnsignedActions}
               />
             </Tabs.Tab>
+            <Tabs.Tab>
+              Unsigned Checklists{' '}

```

Comment:
Please remove, its some prettier auto thing
```suggestion
              Unsigned Checklists

```